tests/test_security_stop_report.py

```
@@ -0,0 +1,69 @@
+from typing import Any, Dict
+import html
+
+# Remove pytest-asyncio dependency to avoid CI issues
+import pytest
+from swarm.api.routes.runs_control import _write_stop_report
+from swarm.api.routes.runs_models import StopReportInfo
+
+def test_write_stop_report_sanitization():
+    """Verify that _write_stop_report sanitizes inputs to prevent Markdown/HTML injection."""
+
+    async def run_test():
```

Ruff (0.15.0)

[warning] Missing return type annotation for private function `run_test`

Add return type annotation: `None`

(ANN202)